specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -240,29 +255,14 @@
           },
           {
             "$ref": "#/parameters/SourceControlIdParameter"
-          },
-          {
-            "$ref": "#/parameters/SourceControl"
           }
         ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/SourceControl"
-            }
-          },
-          "201": {
-            "description": "Created",
-            "schema": {
-              "$ref": "#/definitions/SourceControl"
-            }
+            "description": "OK"
           },
-          "default": {
-            "description": "Error response describing why the operation failed.",
-            "schema": {
-              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
-            }
+          "204": {
+            "description": "No Content"
           }
```

Comment:
Why both 200 and 204 responses? It should have either of them, not both.

Reply:
We return `Warning`s as part of `200` response. `204` will not contain any warnings.